presto-main/src/main/java/com/facebook/presto/execution/SqlQueryManager.java

```diff
@@ -120,10 +124,14 @@
 
     private final SessionPropertyManager sessionPropertyManager;
 
+    private final InternalNodeManager internalNodeManager;
+
     private final Map<Class<? extends Statement>, QueryExecutionFactory<?>> executionFactories;
 
     private final SqlQueryManagerStats stats = new SqlQueryManagerStats();
 
+    private final AtomicBoolean acceptQueries = new AtomicBoolean(false);
```

Comment:
Most of the time we don't pass the default values to constructors as far as I can see. For this particular class if you do a quick search you will see that most usages follow this.